testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/conftest.py

```diff
@@ -243,8 +246,6 @@ def params_from_base_suite_setup(request):
             logging_helper.fetch_and_analyze_logs(cluster_config=cluster_config, test_name=request.node.name)
             raise
 
-    cluster.reset(sg_config)
```

Comment:
Why is this removed?

Reply:
It's not needed as we reset cluster in test itself. Also cluster reset is just after provisioning; doesn't make any sense.